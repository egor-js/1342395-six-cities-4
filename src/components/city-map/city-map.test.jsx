import React from "react";
import renderer from "react-test-renderer";
import CityMap from "./city-map.jsx";
import testData from "../../mocks/test-data.js";

describe(`city map`, () => {
  it(`snapshot test city map`, () => {
    const div = global.document.createElement(`div`);
    div.setAttribute(`id`, `map`);
    global.document.body.appendChild(div);
    const tree = renderer
      .create(
          <CityMap
            places = {testData}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
