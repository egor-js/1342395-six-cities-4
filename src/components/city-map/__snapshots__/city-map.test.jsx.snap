// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`city map snapshot test city map 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
